spec.md

```diff
@@ -164,49 +205,105 @@ echo "SPDX-License-Identifier: MIT" > file.txt
 # REUSE-IgnoreEnd
 ```
 
-### In-line snippet comments
+### REUSE.toml
 
-If a copyright and/or licensing info is to apply only to a certain snippet
-instead of the whole file, SPDX snippet tags SHOULD be used (as defined in [SPDX
-Specification, Annex H](https://spdx.github.io/spdx-spec/v2.3/file-tags/#h3-snippet-tags-format)).
+Copyright and Licensing Information MAY be associated with a file through a
+`REUSE.toml` file. The intended use case of this method is large directories where
+including a comment header in each file (or in `.license` companion files) is
+impossible or undesirable.
 
-Such an annotated snippet block MUST start with `SPDX-SnippetBegin` to mark its
-beginning and end with `SPDX-SnippetEnd` to mark the snippet's end.
+A `REUSE.toml` file MAY be located in any directory, and can cover files that are
+within its directory or deeper. You MAY have multiple `REUSE.toml` files in
+different directories.
 
-Do note that SPDX snippet tags MUST start with `SPDX-Snippet`, meaning that the
-correct copyright notice in a snippet is `SPDX-SnippetCopyrightText`.
+The `version` key (REQUIRED) MUST have an integer value representing the schema
+version of the file. This specification describes version 1 of `REUSE.toml`.
 
-Example:
+Each `[[annotations]]` table represents an association of Copyright and
+Licensing Information to zero or more Covered Files. It has the following keys:
 
-```
-# SPDX-SnippetBegin
-# SPDX-License-Identifier: MIT
-# SPDX-SnippetCopyrightText: 2022 Jane Doe <jane@example.com>
+- `path` (REQUIRED), a string or list of strings representing paths. A path
+  SHOULD resolve to one or more Covered Files relative to the `REUSE.toml`
+  file's directory. A path MAY use globbing to match several Covered Files in a
+  single expression. These are the globbing and matching rules:
 
-{$snippet_code_goes_here}
+  - `*` matches everything except slashes.
```

Comment:
Rather then naming it slashes, we might make explicit that we mean path separators?

Reply:
Added a clarification note to be more specific. Also explicitly added that FORWARD slash === path separator. We had an issue a while ago where someone tried to use Windows slashes in `.reuse/dep5`.